GDJS/Runtime/runtimegame.ts

```diff
@@ -888,6 +889,10 @@ namespace gdjs {
         this._hasJustResumed = false;
         this._renderer.startGameLoop((lastCallElapsedTime) => {
           try {
+            if (this._isDisposed) {
+              return false;
+            }
```

Comment:
This check should not exist, you should stop gameLoop before disposing of the renderer.

Reply:
done